### ISSUE TITLE
mouse cursor hide

### ISSUE THREAD

**Issue by alvgtrujillo** [microsoft/microsoft-ui-xaml#8069]
### Describe the bug

I want to hide the mouse cursor like this in uwp
Window.Current.CoreWindow.PointerCursor = null;

Now i use this in winui :
this.ProtectedCursor = null;

but i cant get the cursor hide.

### Steps to reproduce the bug

 private void canvas_PointerEntered(object sender, PointerRoutedEventArgs e)
        {
            //Window.Current.CoreWindow.PointerCursor = null;
          
          
        }

### NuGet package version

WinUI 3 - Windows App SDK 1.2.2: 1.2.221209.1

### Windows version

Windows 10 (21H2): Build 19044

**Comment by levinli303** [comment]
It seems even using Win32 API SetCursor and ShowCursor cannot hide the cursors in WinUI3.

**Comment by castorix** [comment]
> It seems even using Win32 API SetCursor and ShowCursor cannot hide the cursors in WinUI3.

Yes, I had posted a workaround in https://github.com/microsoft/microsoft-ui-xaml/discussions/7872

**Comment by levinli303** [comment]
@castorix  thanks for the info. if for the purpose of hiding only, it seems we can simply

```
GLView().ProtectedCursor(InputSystemCursor::Create(Input::InputSystemCursorShape::Arrow)); // Since it might be nullptr
GLView().ProtectedCursor().Close();
```
